src/epub/text/farmer.xhtml

```diff
@@ -60,7 +60,7 @@
 				<p>“Hello, Paul,” Johnny McCord said. “Crisis is right. Those jerks down south let a clan of Tuareg, complete with a few thousand goats, camels and sheep through. They’ve been grazing a week or more in my west four hundred.”</p>
 				<p>“Good grief.” Paul grimaced. “At least that’s one thing we don’t have to worry about. They never get this far up. How’d it happen?”</p>
 				<p>“I don’t know, but I’m going to find out. I haven’t seen the mess yet, but it’s certain to wreck that whole four hundred. Have you ever seen just one goat at work on the bark of three-year transplants?”</p>
-				<p>Paul shuddered sympathetically. “Look, Johnny,” he said. “The reason I called you. There’s an air-cushion Land Rover coming through. She just left.”</p>
+				<p>Paul shuddered sympathetically. “Look, Johnny,” he said. “The reason I called you. There’s an aircushion Land Rover coming through. She just left.”</p>
```

Comment:
I think we should basically keep all the dashes that were removed in this PR, except any that were adjusted by modernize-spelling. Most of these words are in M-W with a space, or they are unusual enough where removing the dash would be less clear.

Reply:
All of these were as a result of comparing dashes with the existing stories. I have put most of them back and then changed the other stories to match. Anything in MW I left as is.